src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/RelationsTestCases.java

```diff
@@ -59,4 +55,33 @@ public void relationTestUsingTextAndDisambiguate() {
 		assertEquals("Angular.js", relations.getRelations().get(0).getSubject().getText());
 		assertTrue(relations.getUrl().isEmpty());
 	}
+	
+	private SAORelationsRequest buildTestRequest(int testType) {
```

Comment:
the used of an int value to modify the behavior of a methods in this way is a bat practice, refactor the methods, a better solution would be to create a constructor for SAORelationsRequest and them have 3 methods with clear names or allow in one of them to pass the disambiguate parameter.